axelrod/strategies/titfortat.py

```diff
@@ -636,3 +636,31 @@ def strategy(self, opponent: Player) -> Action:
 
         # Otherwise play previous move
         return self.history[-1]
+
+@FinalTransformer(D, name_prefix=None)
+class Alexei(Player):
+    """
+    Plays similar to Tit-for-Tat, but always defect on last turn.
+    
+    Names:
+
+    - From http://lesswrong.com/lw/7f2/prisoners_dilemma_tournament_results/    
```

Comment:
Take a look at how `Prison` is written in the bibliography. Let me know if you'd like a hand.